guides/common/modules/con_scenarios-for-foreman-on-aws.adoc

```diff
@@ -0,0 +1,84 @@
+[id="scenarios-for-{project-context}-on-aws"]
+= Scenarios for {Project} on AWS
+
+There are several deployment scenarios for {Project} on Amazon Web Services (AWS):
+
+* One-region setup
+* Connecting on-premises and AWS region
+* Connecting different regions
+
+.Scenario 1: One-region setup
```

Comment:
Can you drop "Scenario 1/2/3" and "Option 1/2/3" from, well, everywhere? :D Again, maintenance risk.